code/game/objects/effects/meteors.dm

```diff
@@ -334,6 +345,107 @@ GLOBAL_LIST_INIT(meteors_ops, list(/obj/effect/meteor/goreops)) //Meaty Ops
 	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/blood/xeno(T)
 
+/obj/effect/meteor/meaty/ling
+	name = "suspicous meaty ore"
+	icon_state = "meateor"
+	hits = 1
+	heavy = FALSE
+	var/mob/ling_inside
+	dropamt = 1
+	hitpwr = EXPLODE_LIGHT
+
+/obj/effect/meteor/meaty/ling/examine(mob/user)
+	. = ..()
+	if(ling_inside == user && istype(user, /mob/dead/observer))
+		. += "<span class='changeling'>Right now it's us inside.</span>"
+
+/obj/effect/meteor/meaty/ling/Destroy()
+	var/turf/T = get_turf(src)
+	..()
```

Comment:
```suggestion
	..()
```
please refactor this to do all of this stuff before we return to the parent function. Also you need to return the qdel hint